src/view/frontend/web/js/search-navigation.js

```diff
@@ -13,4 +13,8 @@ define(['factfinder', 'mage/url'], function (factfinder, url) {
     function isSearchResultPage() {
         return window.location.href.indexOf(redirectPath) >= 0;
     }
+
+    function isLoadingPreviousPages() {
+        return event.productsPerPage >= factfinder.communication.EventAggregator.currentSearchResult.paging.resultsPerPage;
```

Comment:
`event` is missing ?

Reply:
thx for catch. Anyway, it couldn't stay like this, so i need to discuss the way of detecting request sent for records from previous pages when restoring scroll position

Reply:
changed to `before-search` discussed yesterday